docs/project_governance.rst

```diff
@@ -5,19 +5,90 @@ Project Governance
 
 This document describes how the QAX Project is managed. The primary topics are:
 
-* The membership of the **QAX Steering Committee**.
-* The decision process related to **QAX Project changes**.
-* The handling of **code changes**.
+* QAX Steering Committee
+    * Role
+    * Membership
+    * Meetings
+* Project work
+    * Definition and process
+    * Decision process
+    * Ad Hoc Meetings
+    * Quality control
+    * Risk management
+    * Problem Resolution
+    * Communications
+    * Principles of collaboration
 
-In brief, the QAX Steering Committee votes on proposals created as
-`GitHub tickets <https://github.com/ausseabed/qax/issues>`_.
-These proposals are available for review for at least three business days.
-A single veto is sufficient to delay proposal acceptance although ultimately
-a majority of members can pass a proposal.
-All not-minor code changes require at least 1 reviewer.
 
 Steering Committee
 ------------------
+Roles
+^^^^^
+The Steering Committee:
+
+* Oversees the management and direction of the QAX project
+* Votes on proposals created as GitHub issues
+* Manages roles within the GitHub repository (e.g. reviewer, owner, etc)
+* Will elect a Chair from the committee
+* Are responsible for communicating project intentions to the broader community
+
+Chair:
+
+* Responsible for leading and organizing meetings
+* Managing GitHub membership and roles
+* Monitoring actions arising from committee meetings
+* Maintain decision registry
+* Will retain the role for 1 year (??)
+* Is elected annually by the Steering Committee (??)
+
+
+Bootstrapping
+^^^^^^^^^^^^^
+
+Lachlan Hurst (FrontiersSI), Justy Siwabessy (Geoscience Australia), Zzzz Wwwww (Kkkkk), Tyanne Faulkes (NOAA, OCS) and
```

Comment:
To keep a proper balance in the committee, I believe that "Zzzz Wwwww (Kkkkk)" should be an AusSeabed  folk.